bundles/org.openhab.binding.luftdateninfo/src/main/java/org/openhab/binding/luftdateninfo/internal/handler/BaseSensorHandler.java

```diff
@@ -24,7 +26,9 @@
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
 import org.eclipse.smarthome.core.types.Command;
 import org.eclipse.smarthome.core.types.RefreshType;
+import org.joda.time.DateTime;
```

Comment:
Joda found its replacement in Java 8. You could use `ZonedDateTime`.

Reply:
Sorry for the rework. But after my observations during running this binding it was necessary:

- REST API calls of luftdateninfo are taking sometimes a lot of time. Many warnings were printed in the logs which isn't covenient. Now http timeout is increased and details with timestamp are delivered via "updateStatus" so they are shown in the UI

- UoM wasn't properly defined in sitemap and items. This is corrected

- Translations on channel-type were missing

I think it was worth to apply these corrections before a second reviewer starts his work.